release/requirements_buildkite.in

```diff
@@ -15,3 +15,6 @@ pybuildkite
 PyGithub
 requests
 retry
+
+# Below are requirements only used by ray_ci
+colorama; platform_system == "Windows"
```

Comment:
colorama is a requirement of anyscale too so maybe just remove this file

Reply:
also to make it works on windows, can you add `tzdata` as a requirement here

Reply:
added tzdata and removed colorama.